tsconfig.dist.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "./tsconfig.json",
+  "exclude": ["dist", "**/*.test.ts"]
```

Comment:
I don't think you have to exclude dist if it's configured as the outdir, but I may be wrong about this (I haven't used typescript + javascript much).

And yeah, two tsconfig files is p much the only way of doing this, which is kinda lame but what can we do but play the hand dealt us. :man_shrugging:

Reply:
Typescript complains about overwriting input files without the inclusion of `dist` in the `exclude`